charts/ehrenamt-justiz/templates/tests/test-connection.yaml

```
@@ -0,0 +1,20 @@
+  labels:
+    {{- include "getLabels" $data | trim | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+spec:
+  containers:
+    - name: wget
+      image: busybox
+      command: ['wget']
+      args: ['{{ include "getFullname" $data }}:8080/actuator/health']
+  restartPolicy: Never
+{{- end }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)